extensions/amp-audio/1.0/test/test-component.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '#preact';
+import {Audio} from '../component';
+import {mount} from 'enzyme';
+
+describes.sandboxed('Audio preact component v1.0', {}, () => {
+  it('should load audio through attribute', () => {
+    const wrapper = mount(
+      <Audio src="audio.mp3" style="height: 30px; width: 300px" />
```

Comment:
ah, and please also give style as an object:
```suggestion
      <Audio src="audio.mp3" style={{height: '30px', width: '300px'}} />
```